prime-router/src/main/kotlin/history/azure/DatabaseDeliveryAccess.kt

```diff
@@ -141,4 +144,46 @@ class DatabaseDeliveryAccess(
             query.fetchInto(DeliveryFacility::class.java)
         }
     }
+
+    /**
+     * Get a list of matching delivery facilities for a set of [reportIds]
+     */
+    fun fetchBulkDeliveryFacilities(
```

Comment:
Can we add a comment tagging this with this ticket https://github.com/CDCgov/prime-reportstream/issues/9272?

Reply:
Done!